example/gluon/wavenet/data_loader.py

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License
+"""
+Description : Set Dataset module for Wavenet
+"""
+import os
+import numpy as np
+from mxnet import nd
+from scipy.io import wavfile
+from utils import encode_mu_law
+# pylint: disable=invalid-name, too-many-arguments
+
+def load_wav(file_nm):
+    """
+    Description : load wav file
+    """
+    fs, data = wavfile.read(os.getcwd()+'/dataset/'+file_nm)
+    return  fs, data
```

Comment:
return directly instead of using fs, data?

Reply:
fix code https://github.com/apache/incubator-mxnet/pull/13735/commits/32a3b6eb2b53f27a5bddbfd130ac2e357877475d